src/output.rs

```diff
@@ -792,11 +792,12 @@ impl<W: io::Write> Formatter<W> for Rpsl {
 
 //------------ Summary -------------------------------------------------------
 
-struct Summary;
+/// Output only a summary.
+pub struct Summary;
 
-impl<W: io::Write> Formatter<W> for Summary {
-    fn header(
-        &self, _snapshot: &PayloadSnapshot, metrics: &Metrics, target: &mut W
+impl Summary {
+    pub fn write(
```

Comment:
is `write` still used somewhere? Looks like `log` + `produce_header` replaced it entirely.

Reply:
Uhm, indeed. Forgot to delete it. Fixed.